platform-coop-toolkit/coop-map-directory-index
# feat: add organization phone (resolves #73)

## Description

Adds organization phone field.

## Steps to test

1. Check out branch.
2. Run migrations: `./manage.py migrate`
3. Log in to admin.

**Expected behavior**: Organizations now include a "Phone" field.

## Additional information

Not applicable.

## Related issues

- Requires #66.
- Requires #77.
- Resolves #73.
